tests/test_worker/test_categorization.py

```diff
@@ -1,13 +1,23 @@
 import pytest
-from sourcemash.categorize import categorizeItem
+from sourcemash.categorize import categorize_item
 from collections import Counter
 
 class TestCategorize:
 
     def test_categorize_item(self, ebolaItem):
-		category_dict = Counter()
-		category_dict.update(ebolaItem.title.split())
-		
-		(cat1, cat2) = categorizeItem(ebolaItem.title, ebolaItem.text, category_dict)
+        category_dict = Counter()
+        category_dict.update(ebolaItem.title.split())
+        
+        (cat1, cat2) = categorize_item(ebolaItem.title, ebolaItem.text, category_dict)
 
-		assert (cat1, cat2) == ("Ebola", "West")
\ No newline at end of file
+        assert (cat1, cat2) == ("Ebola", "West")
+
+    def test_empty_categories(sef, oftheItem):
+        category_dict = Counter()
```

Comment:
Instead of manually making the category dict in the test, you should probably create a scraper class and run the reset dictionary function on it.